Parsers/ASimAuditEvent/Parsers/vimAuditEventGithub.yaml

```diff
@@ -0,0 +1,380 @@
+Parser:
+    Title: Audit Event ASIM filtering parser for Microsoft Github audit events
```

Comment:
there are comments in the old PR that were not applied. Can you please review and resolve?
https://github.com/Azure/Azure-Sentinel/pull/10207